backend/src/main/java/com/backend/Controller/BookController.java

```diff
@@ -40,81 +39,115 @@ public ResponseEntity<Book> getBookById(@PathVariable Long id) {
 	}
 
 	@PostMapping
-	public ResponseEntity<Book> createBook(@RequestBody Book book) {
-		if (book.getImage() != null) {
-			try {
-				book.setImage(Base64.getDecoder().decode(new String(book.getImage())));
-			}
-			catch (IllegalArgumentException e) {
-				logger.error("Invalid Base64 input for image", e);
-				return ResponseEntity.badRequest().body(null);
-			}
+	public ResponseEntity<String> createBook(@RequestParam("title") String title,
```

Comment:
De ce ai inlocuit body cu query parameters? Ar trebui sa trimitem datele folosind body

Reply:
Am inlocuit cu model attribute, nu putem folosi RequestBody pt ca nu putem trimite fisierul in json